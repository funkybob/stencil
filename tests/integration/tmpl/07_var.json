{
    "top": "hat",
    "map": {"key": "value"},
    "sequence": ["zero", "one", "two", "three"]
}
