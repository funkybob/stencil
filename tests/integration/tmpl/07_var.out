
hat
value
two
literal
123
